index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+</head>
+<body>
+    <h2><strong>📘 The Code Magazine</strong></h2>
+    <h3>The Basic Language of the Web: HTML</h3>
+    <img src="img/laura-jones.jpg">
+    <p>Posted by <strong>Laura Jones</strong> on Monday, June 21st 2027</p>
+    <img src="img/post-img.jpg" height="200" width="500">
+    <p>All modern websites and web applications are built using three fundamental technologies: HTML, CSS and JavaScript. These are the languages of the web.
+        In this post, let's focus on HTML. We will learn what HTML is all about, and why you too should learn it.</p>
+    <h3><strong>What is HTML?</strong></h3>
```

Comment:
WHy you wrap all headings content into `<strong>` ? 
There is no need since they are already headings and browser considers them important